README.md

```diff
@@ -7,13 +7,33 @@ This repository contains a list of NZBGet extensions, which is used by the Exten
 1. Make your extension compatible with NZBGet spec v23+. 
 [Documentation](https://github.com/nzbgetcom/nzbget/blob/develop/docs/extensions/EXTENSIONS.md).
 
-2. Once your extension is ready, please contact NZBGet maintainers to get it added into the extension manager 
-(we'll create a project for you - `https://github.com/nzbgetcom/Extension-YourExtensionName/`).
+2. Make sure that yours extension repository contains the required and possibly optional workflows:
 
-3. Create a fork of that repository, add your extension to it, and make a Pull request.
+Mandatory:
+- Prospector Python Static Analysis check
+- Manifest check
 
->You can use any of nzbgetcom extension as example, 
-e.g [FakeDetector](https://github.com/nzbgetcom/Extension-FakeDetector).
+Optional:
+- Tests check (availability of tests is highly recommended)
```

Comment:
(availability of tests is highly recommended) rephrase this, reads a bit odd for me.

Reply:
-> Having tests for your extension is highly recommended